src/main/java/core/basesyntax/dao/impl/SmileDaoImpl.java

```diff
@@ -12,16 +14,44 @@ public SmileDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Smile create(Smile entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            session.flush();
+            session.clear();
```

Comment:
redundant black line
also, session is flushed automatically inside `transaction.commit()`